go.mod

```diff
@@ -14,10 +14,7 @@ require (
 	github.com/google/uuid v1.1.2
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/klauspost/compress v1.10.8
```

Comment:
Curious why is the mod and sum file changing? Can these changes be done in a separate PR?

Reply:
Let me recheck again :)

Reply:
synced with master branch